bootstrap_cfn/config.py

```diff
@@ -91,10 +91,14 @@ def process(self):
     def base_template(self):
         from bootstrap_cfn import vpc
         t = Template()
-
-        t.add_mapping("AWSRegion2AMI", {
-            "eu-west-1": {"AMI": "ami-00d88f77"},
-        })
+        if 'os' in self.data['ec2'] and self.data['ec2']['os'] == 'windows2012':
```

Comment:
If we're adding support to choosing the AMI, we might want to make this a bit more expansive. i.e